base/lsif-server/lsif-server.Deployment.yaml

```
@@ -63,6 +63,17 @@ spec:
           name: lsif-storage
       securityContext:
         runAsUser: 0
+      # lsif-server incorrectly previously ran as `root` instead of the
+      # `sourcegraph` user. We use this init container (ran as root) to fix
```

Completely trivial grammatical nit.
```suggestion
      # `sourcegraph` user. We use this init container (run as root) to fix
```